tests/singlepage/views/singlepage.js

```diff
@@ -14,13 +14,19 @@ function __setLocation(url) {
 	window.history.replaceState({}, undefined, url);
 }
 
-function __triggerClickEvent(self) {
+function __triggerClickEvent(self ,options) {
 	var isDefaultPrevented = false;
 	self.view.$el.on('click', 'a', function(event) {
 		isDefaultPrevented = event.isDefaultPrevented();
 		event.preventDefault();
 	});
-	self.view.$el.find('a').trigger(new $.Event('click'));
+
```

Comment:
You don't need to separate here. It doesn't matter if the second parameter is defined or undefined for the options parameter of the `$.Event` constructor.